src/makeInfinityAdder.js

```diff
@@ -7,6 +7,24 @@
 
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+  let counter = 0;
+
+  const adder = (num) => {
+    if (num === undefined && counter === 0) {
+      counter++;
+
+      return sum;
+    } else if (counter > 0) {
+      return 0;
+    }
```

Comment:
counter variable is redundant here
you shouldn't store the num of function calls
if num = undefined - you should return sum and reset this value
if num != undefined - you should add num to the sum and return the function